nautobot_version_control/templates/nautobot_version_control/commit.html

```diff
@@ -1,4 +1,4 @@
-{% extends 'base.html' %}
+{% extends 'generic/object_detail.html' %}
```

Comment:
In addition to this change, need to review whether any or all of the contents of `block header` are now redundant, and if they can be removed to simplify the template.

Additionally, `block content` should be updated to only override `block content_[left|right|full_width]_page` as appropriate.

Same comment on the other template files below.